guides/source/action_controller_overview.md

```diff
@@ -10,7 +10,7 @@ After reading this guide, you will know:
 * How to follow the flow of a request through a controller.
 * How to restrict parameters passed to your controller.
 * How and why to store data in the session or cookies.
-* How to work with filters to execute code during request processing.
+* How to work with callbacks to execute code during request processing.
```

Comment:
perhaps instead of replacing `filters` with `callbacks` it should be `action callbacks`? Here and elsewhere